### ISSUE TITLE
Fail to login

### ISSUE THREAD

**Issue by hunhun010164** [microg/GmsCore#2560]
I have already updated to the latest version v0.3.2.240913

Then I logged out and tried logging in again. But another issue occurred...
![Weixin Image_20240925191414](https://github.com/user-attachments/assets/ef581186-80be-4022-8500-cceef7e97053)

For more detailed info, please refer to https://github.com/microg/GmsCore/issues/2548

Could anyone help me? Thanks.

**Comment by ILoveOpenSourceApplications** [comment]
This issue is similar to #2541.

**Comment by mar-v-in** [comment]
What device are you using? If you are using a device that does not come with Google pre-installed (like Huawei) you might need to make sure you have:
- "Authenticate with device registration" **disabled** in microG Settings -> Google Accounts
- "Strip device name for authentication" **enabled** in microG Settings -> Google Accounts

**Comment by trd-warren** [comment]
@mar-v-in

Hello. How about support for `-lh` variant?  Wasn't included in the last two releases. Only that variant worked for my device.

Huawei Nova 8i.

**Comment by hunhun010164** [comment]
> What device are you using? If you are using a device that does not come with Google pre-installed (like Huawei) you might need to make sure you have:
> 
> * "Authenticate with device registration" **disabled** in microG Settings -> Google Accounts
> * "Strip device name for authentication" **enabled** in microG Settings -> Google Accounts

Yes, it does not come with Google pre-installed. See the screenshot below
[
![Weixin Image_20241001223332](https://github.com/user-attachments/assets/78ade4d5-ef33-43f7-bef5-a8760725bdb2)
](url)

I can login after taking your advice. But youtube still not working. Saying that I'm not logged in.
![youtube](https://github.com/user-attachments/assets/663ec151-4bf2-4f14-ab49-1f6095d78b61)

**Comment by hunhun010164** [comment]
> This issue is similar to #2541.

Thank you. I can login now. But youtube still not working.

**Comment by 0JellyDice0** [comment]
> What device are you using? If you are using a device that does not come with Google pre-installed (like Huawei) you might need to make sure you have:
> 
> * "Authenticate with device registration" **disabled** in microG Settings -> Google Accounts
> * "Strip device name for authentication" **enabled** in microG Settings -> Google Accounts

It works on Windows Subsystem for Android, thanks mate
